contracts/utils/OperatorSignedHashes.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.19;
+pragma solidity ^0.8.30;
```

Comment:
Changes in this contract are needed, because it's becoming part of the contract that is a proxy - thus, domain separator (for signatures) can't be a constant anymore.